Assets/Scripts/Weapons/Data/WeaponData.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) TigardHighGDC
+// SPDX-License SPDX-License-Identifier: Apache-2.0
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[CreateAssetMenu(fileName = "New Weapon", menuName = "ScriptableObject/WeaponData")]
+public class WeaponData : ScriptableObject
+{
+    [Header("Stats")]
+    public float ReloadSpeed;
+    public float BulletSpeed;
+    public float BulletPerSecond;
+    public int BulletPerTrigger;
+    public int AmmoCapacity;
+    public float Spread;
+    public float Knockback;
+    public float BulletDespawn;

```

Comment:
```suggestion
    public float BulletDespawnTimeSeconds;
```

Is it correct that this float keeps track of seconds?

Reply:
This float keeps track of seconds. I'll make the changes to make sure the code doesn't break.